cl/search/models.py

```diff
@@ -545,6 +556,36 @@ class Docket(AbstractDateTimeModel, DocketSources):
         blank=True,
         db_index=True,
     )
+    office_code = models.CharField(
+        help_text="A one digit statistical code (either alphabetic or numeric) "
+        "of the office within the district. In this "
+        "example, 2:07-cv-34911-MJL, the 2 preceding "
+        "the : is the office code.",
+        max_length=1,
+        blank=True,
+    )
+    case_type = models.CharField(
+        help_text="Case type, e.g., civil (cv), magistrate (mj), criminal (cr), "
+        "petty offense (po), and miscellaneous (mc). These codes "
+        "can be upper case or lower case, and may vary in number of "
+        "characters.",
+        max_length=5,
+        blank=True,
```

Comment:
Do we want an index on this? If so, can you put it in the meta class?

Reply:
good question - @ERosendo and I had a long back and forth before we removed all the indexes I had originally proposed.

Reply:
@mlissner, @ERosendo and I were discussing whether the low cardinality of the field would actually hurt in this situation.  Considering CR and CV MJ and BK are so much larger than other case types we left it off.  But we would be open to hearing your thoughts.

Reply:
Cool.